.github/workflows/remove-issue-labels.yaml

```diff
@@ -1,12 +1,14 @@
-name: Remove Labels
-
-on:
-  issues:
-    types: [closed]
-
-jobs:
-  remove_label:
-    runs-on: ubuntu-latest
-    steps:
-      - name: Remove label
-        run: gh issue edit ${{ github.event.issue.number }} --remove-label "status:triage"
+name: Remove Labels
+
+on:
+  issues:
+    types: [closed]
+
+jobs:
+  remove_label:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Remove label
+        run: gh issue edit ${{ github.event.issue.number }} --remove-label "status:triage"
+        env:
```

Comment:
these last two lines are the only lines I actually changed; github is rendering the whole file though because it contained carriage returns and I removed them all in favor of standard newlines